### ISSUE TITLE
3.1.0 gulping error

### ISSUE THREAD

**Issue by drjonnicholson** [swagger-api/swagger-ui#3485]
Laravel 5.3 site, deployed using Envoyer to a Forge server. Includes swagger-ui-dist@3.1.0.

3.1.0 causes my Gulp build to fail on the server, but apparently not locally.

Error is:
```bash
Error: Cannot find module './absolute-path.js' from '**path**/node_modules/swagger-ui-dist'
```

Suspect it's an issue in 3.1.0 as I successfully deployed the morning prior to 3.1.0 release and after it's failing. Reverted to latest 3.0.* release and it works again.

**Comment by shockey** [comment]
@drjonnicholson, thanks for the heads up, this is probably on our end - we ran the `swagger-ui-dist` release a little differently this time, there may have been a problem in the build.

I'll look into this now - expect to hear from me shortly!

**Comment by shockey** [comment]
@drjonnicholson, I've just released `swagger-ui-dist@3.1.1`. It appears to have solved the problem.

Please confirm that the issue is fixed for you at your earliest convenience!

**Comment by drjonnicholson** [comment]
Sorry for the delay. Yes this resolved the issue! Thanks!
